cmd/gnodev/mod.go

```diff
@@ -12,38 +12,62 @@ import (
 	"github.com/gnolang/gno/pkgs/gnolang/gnomod"
 )
 
-func newModCmd() *commands.Command {
-	return commands.NewCommand(
+type modDownloadCfg struct {
+	remote string
+}
+
+var defaultModDownloadOptions = &modDownloadCfg{
+	remote: "test3.gno.land:36657",
+}
```

Comment:
If you don't share this configuration between files in this package, you can safely drop it and just have the default option be set when you are registering flags in `modDownloadCfg.RegisterFlags()`

Reply:
Done 356b1fe8e2bfcfc5b484f7f2edd845919312a55a